src/main/resources/com/suse/matcher/rules/drools/InputAugmenting.drl

```diff
@@ -38,3 +38,73 @@ rule "generatePenaltyGroups"
         insert(new PenaltyGroup(penaltyGroupId, $guestId));
 end
 
+// aggregate 2 equal (same properties, same product set) subscriptions into a single one
+rule "aggregateEqualStackableSubscriptions"
+    agenda-group "InputAugmenting"
+    when
+        $s1: Subscription(ignored == false, stackable == true, aggregationSubscriptionId == null || aggregationSubscriptionId == id)
+        $s2: Subscription(
+            ignored == false,
+            stackable == true,
+            id > $s1.id,
+            id != aggregationSubscriptionId, // rule out subs that already have been aggregated into other subscription
```

Comment:
This comment should be `// rule out subs that already aggregate other subscription` - it's changed in one of the newer commits.